simple_history/tests/tests/test_models.py

```diff
@@ -2841,3 +2844,132 @@ def test_historic_to_historic(self):
         )[0]
         pt1i = pt1h.instance
         self.assertEqual(pt1i.organization.name, "original")
+
+
+class HistoricOneToOneFieldTest(TestCase):
+    """
+    Tests chasing OneToOne foreign keys across time points naturally with
+    HistoricForeignKey.
+    """
+
+    def test_non_historic_to_historic(self):
+        """
+        Non-historic table foreign key to historic table.
```

Comment:
Are these comments referring to foreign key correct? Should any be reworded to one to one field?

Reply:
They should be